From: Jan Engelhardt <jengelh@inai.de>
Date: 2014-02-05 00:33:39.300262391 +0100

build: stop misplacing documentation

docdir must not be statically set as it would override what the
user gave to ./configure --docdir=...

---
 doc/Makefile.am |    1 -
 1 file changed, 1 deletion(-)

Index: 4ti2-1.6/doc/Makefile.am
===================================================================
--- 4ti2-1.6.orig/doc/Makefile.am
+++ 4ti2-1.6/doc/Makefile.am
@@ -1,4 +1,3 @@
-docdir = $(pkgdatadir)/doc
 
 doc_DATA = 4ti2_manual.pdf
 
